libqtile/backend/wayland/window.py

```diff
@@ -153,6 +153,11 @@ def __init__(self, core: Core, qtile: Qtile, surface: S):
         # attribute optional to avoid the destroy().
         self.ftm_handle: ftm.ForeignToplevelHandleV1 | None = None
 
+        self.base_x: int | None = None
+        self.base_y: int | None = None
+        self.base_width: int | None = None
+        self.base_height: int | None = None
```

Comment:
I think you can include this is `base.Window.__init__` as that's called at the start of both the Wayland and X11 window definitions.

Reply:
Done.